### PR TITLE
Add support for children to Snackbar

### PR DESCRIPTION
**Pull request by dwkaminsky**
## Description
Adds Support for Children to Snackbar Component

image

### CODE REVIEW

File: src/lib/components/Snackbar.react.js [hunk 1]
```diff
@@ -41,6 +42,8 @@ type Props = {
   },
   /** CSS class name of the root element */
   className?: string,
+  /** Elements to render inside the snackbar. Note that this does not work along message and actions */
+  children?: Node,
```
Comment (coralvanda) [inline src/lib/components/Snackbar.react.js:46]: Phrasing here is a bit odd.

File: src/lib/components/Snackbar.react.js [hunk 2]
```diff
@@ -109,14 +114,15 @@ export default class Snackbar extends Component<Props, State> {
...
-    const {action, actionStyles, autoHideDuration, className, id, message, style } = this.props;
+    const {action, actionStyles, autoHideDuration, className, children, id, message, style } = this.props;
 
     return action ? (
       <div id={id}>
         <MuiThemeProvider muiTheme={getMuiTheme(lightBaseTheme)}>
           <MuiSnackbar
             autoHideDuration={autoHideDuration}
             className={className}
+            children={children}
```
Comment (coralvanda) [inline src/lib/components/Snackbar.react.js:125]: Is there any need for us to check the values of `message` or `children` before attempting to render them? If the user submits both, ignoring the warning from the docs, what happens?

**Review by coralvanda — CHANGES_REQUESTED** [review]

File: src/lib/components/Snackbar.react.js (hunk 2 above)
Comment (dwkaminsky) [reply src/lib/components/Snackbar.react.js:125]: Children will override Message and Actions. I modified the documentation to reflect this as well

File: src/lib/components/Snackbar.react.js (hunk 1 above)
Comment (dwkaminsky) [reply src/lib/components/Snackbar.react.js]: Not sure what I meant here, clearly not correct. I clarified now

**Review by coralvanda — APPROVED** [review]

**Pull request merged**